src/accelerate/optimizer.py

```diff
@@ -63,6 +63,18 @@ def __init__(self, optimizer, device_placement=True, scaler=None):
     def param_groups(self):
         return self.optimizer.param_groups
 
+    @param_groups.setter
+    def param_group(self, param_groups):
+        self.optimizer.param_groups = param_groups
+
+    @property
+    def defaults(self):
+        return self.optimizer.defaults
+
+    @defaults.setter
+    def param_group(self, defaults):
+        self.optimizer.defaults = defaults
```

Comment:
PyTorch optimizers are expected to have thos attributes setable, so I'm obliging.